pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -1334,6 +1334,11 @@ func updatePipelineRunStatusFromChildObjects(ctx context.Context, logger *zap.Su
 	fullEmbedded := cfg.FeatureFlags.EmbeddedStatus == config.FullEmbeddedStatus || cfg.FeatureFlags.EmbeddedStatus == config.BothEmbeddedStatus
 	minimalEmbedded := cfg.FeatureFlags.EmbeddedStatus == config.MinimalEmbeddedStatus || cfg.FeatureFlags.EmbeddedStatus == config.BothEmbeddedStatus
 
+	if cfg.FeatureFlags.EmbeddedStatus == config.MinimalEmbeddedStatus {
```

Comment:
should the RHS of == be `minimalEmbedded` or `config.MinimalEmbeddedStatus` (double checking)

Reply:
Thank you for the checking, I think this should be the `MinimalEmbeddedStatus` which is string 'minimal' in the `config-feature-flags.yaml` because the minimalEmbedded above is a union of `Both` and `Minimal` EmbeddedStatus.

Reply:
Sounds good!